### PR TITLE
update cluster ip notes

### PR DESCRIPTION
**Pull request by philbrookes**
update cluster ip notes

### CODE REVIEW

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 1]
```diff
@@ -100,6 +100,15 @@ the defaults are the safest option.
 The *Wildcard DNS Host* option alters the wildcard DNS host you wish
 to use.
 
+. Check the Cluster IP:
++
+In order to allow communication between your mobile device and the cluster, the
+cluster IP must be in the same as your local machines IP address.
```
Comment (jstaffor) [inline modules/ROOT/pages/installing-mobile-services.adoc]: "must be ~~in~~ the same as your local machines IP address."

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 2]
```diff
@@ -100,6 +100,15 @@ the defaults are the safest option.
 The *Wildcard DNS Host* option alters the wildcard DNS host you wish
 to use.
 
+. Check the Cluster IP:
++
+In order to allow communication between your mobile device and the cluster, the
```
Comment (jstaffor) [inline modules/ROOT/pages/installing-mobile-services.adoc]: ~~In order~~  To allow communication between your mobile device and the cluster

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 3]
```diff
@@ -100,6 +100,15 @@ the defaults are the safest option.
...
+cluster IP must be in the same as your local machines IP address.
++
+i.e. If your workstation has the IP address: "192.168.0.2" and your mobile device
```
Comment (jstaffor) [inline modules/ROOT/pages/installing-mobile-services.adoc]: ~~i.e.~~ For example, if your ...

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 4]
```diff
@@ -100,6 +100,15 @@ the defaults are the safest option.
 The *Wildcard DNS Host* option alters the wildcard DNS host you wish
 to use.
 
+. Check the Cluster IP:
++
+In order to allow communication between your mobile device and the cluster, the
+cluster IP must be in the same as your local machines IP address.
++
+i.e. If your workstation has the IP address: "192.168.0.2" and your mobile device 
+has the IP `192.168.0.3` then you should set the cluster IP to: `192.168.0.2` so
```
Comment (jstaffor) [inline modules/ROOT/pages/installing-mobile-services.adoc]: the IP `192.168.0.3`, ~~then you should~~ set the cluster IP to: `192.168.0.2` so

File: modules/ROOT/pages/installing-mobile-services.adoc [hunk 5]
```diff
@@ -100,6 +100,15 @@ the defaults are the safest option.
...
 to use.
 
+. Check the Cluster IP:
++
+In order to allow communication between your mobile device and the cluster, the
+cluster IP must be in the same as your local machines IP address.
++
+i.e. If your workstation has the IP address: "192.168.0.2" and your mobile device 
+has the IP `192.168.0.3` then you should set the cluster IP to: `192.168.0.2` so 
+that your mobile will be able to access the cluster.
```
Comment (jstaffor) [inline modules/ROOT/pages/installing-mobile-services.adoc]: that your mobile device ~~will be~~ is able to access the cluster.

**Comment by philbrookes** [comment]
@jstaffor made those changes now

**Pull request merged**